js/windowMgr.js

```diff
@@ -129,11 +132,14 @@ function doCreateMainWindow(initialUrl, initialBounds) {
             // removes all existing notifications when main window reloads
             notify.reset();
             log.send(logLevels.INFO, 'main window loaded url: ' + url);
+
+            // Initiate activity detection to monitor user activity status
+            activityDetection.initiateActivityDetection();
```

Comment:
when user does a reload this will get invoked again.  and this will cause your method to be re-invoked.  does this cause any issues?  e.g., does setInternval needed to be canceled and restarted?